packages/scheduler/src/__tests__/SchedulerSetTimeout-test.js

```diff
@@ -21,8 +21,9 @@ describe('SchedulerNoDOM', () => {
   // This is only meant to be used for testing purposes, like with jest's fake timer API.
   beforeEach(() => {
     jest.resetModules();
-    jest.useFakeTimers();
+    jest.useFakeTimers({legacyFakeTimers: true});
     delete global.setImmediate;
+    delete global.MessageChannel;
```

Comment:
`MessageChannel` is now defined in the Node.js environment but the test here specifically targets the `setTImeout` fallback.